.github/workflows/maven-ci.yml

```diff
@@ -0,0 +1,108 @@
+name: OTP CI Build
+
+# TODO: Perhaps reinstate, but doesn't allow for build/test of any branch.
+# On [push, pull_request] causes double-builds when creating PRs.
+# But triggering on push only will miss pull requests from outside authors.
+# The push event's ref is the name of the pushed branch;
+# The pull_request event's branch name is the merge target branch.
+#on:
+#  push:
+#    branches:
+#      - master
+#      - dev-1.x
+#      - dev-2.x
+#      - ibi-dev
+#  pull_request:
+#    branches:
+#      - master
+#      - dev-1.x
+#      - dev-2.x
+#      - ibi-dev
```

Comment:
I think we can remove this commented out branch filtering.

Reply:
@landonreed I'm seeing double builds under GitHub actions as noted in the original comment, perhaps there is something in this?!